package.json

```diff
@@ -14,38 +14,38 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@mantine/core": "^7.2.2",
-    "@mantine/hooks": "^7.2.2",
-    "clsx": "^2.0.0",
-    "eslint": "^8.51.0",
-    "eslint-config-next": "^14.0.1",
```

Comment:
In dependencies we have deps which are needed to run the actual project in production. In devdependencies we have things which are only necessary for development.